ykrt/src/mt.rs

```diff
@@ -1,3 +1,5 @@
+//! Initialise the meta-tracing system.
```

Comment:
This one isn't exactly true. Maybe something like "The main end-user interface to the meta-tracing system"?

Reply:
Hah, I just copied your suggestion from here: https://github.com/softdevteam/yk/pull/226/files#diff-9fa85f3ab0ab08c2bb4337dd17d47605f520c2780158e50e66eff7cd9dc1b29bR1